daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/iterable/TypeIterable.scala

```diff
@@ -79,8 +79,8 @@ private[validation] object TypeIterable {
         Iterator(typ) ++
           iterator(value)
       case EVar(_) | EVal(_) | EBuiltin(_) | EPrimCon(_) | EPrimLit(_) | EApp(_, _) | ECase(_, _) |
-          ELocation(_, _) | EStructCon(_) | EStructProj(_, _) | EStructUpd(_, _, _) |
-          ETyAbs(_, _) =>
+          ELocation(_, _) | EStructCon(_) | EStructProj(_, _) | EStructUpd(_, _, _) | ETyAbs(_, _) |
+          EExperimental(_, _) =>
         ExprIterable.iterator(expr0).flatMap(iterator(_))
```

Comment:
doesn’t `EExperimental` have a type we need to traverse?

Reply:
:+1: